src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue

```diff
@@ -1725,31 +1725,24 @@ export default Vue.extend<
                         ? this.allianceBuildings[intId]
                         : this.buildings[intId];
                     if (!building) return null;
-                    const buildingType =
-                        this.buildingTypes[building.building_type];
-
-                    let bigBuildingType = building.building_type;
-                    if (building.small_building) {
-                        bigBuildingType = parseInt(
-                            (
-                                Object.entries(smallBuildings).find(
-                                    ([, small]) =>
-                                        small === building.building_type
-                                )?.[0] ?? bigBuildingType
-                            ).toString()
-                        );
-                    }
+
+                    const realBuildingType = building.small_building
```

Comment:
Please use at least brackets to maintain a little bit of readability.

Reply:
ESLint removes the brackets so I cannot make this section more readable by adding brackets.

Reply:
Do we really need Eslint? :wink: